docs/kyma/docs/037-CRD.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
```

Comment:
if this is CRD, which btw I think it is a very good idea
this should follow same structure as the other CRD you already created